### PR TITLE
VS Code: set PHP version level to 7.2

### PR DESCRIPTION
**Pull request by kraftbj**
In setting up VS Code, I updated the PHP version to 7.2, which edits the settings.json for the repo. Should we set this as part of the default settings file sent over?

## Proposed changes:

* Set VSC to PHP 7.2

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Jetpack product discussion

n/a

## Does this pull request change what data or activity we track or use?

No.

## Testing instructions:

* Open in VS Code and see if the bottom-right indicates PHP 5.6 when opening a PHP file.

### CODE REVIEW

**Comment by kraftbj** [comment]
Updated to PHP 7.0.

File: .vscode/settings.json [hunk 1]
```diff
@@ -34,5 +34,6 @@
 		"**/jetpack_vendor/**",
 		"**/vendor/**",
 	],
-	"prettier.prettierPath": "tools/js-tools/node_modules/prettier/index.cjs"
-}
\ No newline at end of file
+	"prettier.prettierPath": "tools/js-tools/node_modules/prettier/index.cjs",
+	"php.version": "7.0"
```
Comment (kraftbj) [inline .vscode/settings.json]:
```suggestion
	"php.version": "7.2"
```

**Review by jeherve — COMMENTED** [review]
I'm not exactly sure what this does, and how it's used. Given that we rely on PHP 8.2 for other tooling in the monorepo, maybe that's what we should use here too?

**Comment by kraftbj** [comment]
@jeherve The intent with this is for the PHP version the code supports; not dev tooling.

**Review by jeherve — APPROVED** [review]

**Pull request merged**